Instructions on how to do a blitz release (#796)
blitz-js/blitzjs.com: app/pages/docs/contributing.mdx

pnpm link ./packages/blitz
```

> ## Doing a release {#doing-a-release}

Reviewer: Title Case

> Depending on your maintainer level, you will have the ability to make a
> blitz release. The way it currently works, isn't set in stone so look here
> for latest updates on how the release system works.

Reviewer: Get straight to the goods :) 
```suggestion
```

We use the [changesets](https://github.com/changesets/changesets) package
& a forked version of their GitHub action which can be found here:
[GitHub - blitz-js/changesets-action](https://github.com/blitz-js/changesets-action).

> Essentially every user facing PR should have a changeset. When that PR

Reviewer: ```suggestion
Every user facing PR should have a changeset. When that PR
```

> gets approved and merged to the `main` branch a new PR is either created
> or modified called `Version packages (beta)`. This PR updates each

Reviewer: ```suggestion
gets approved and merged to the `main` branch the "release" PR with name 
`Version packages (beta)` is automatically created or updated. This PR updates each
```
